# Request message type
bool reset
---
# Response message type
bool reset_done
